src/components/enterprise-user-subsidy/data/constants.js

```diff
@@ -5,3 +5,7 @@ export const LICENSE_STATUS = {
 };
 
 export const LOADING_SCREEN_READER_TEXT = 'loading your edX benefits from your organization';
+
+export const enterpriseUserSubsidyQueryKeys = {
```

Comment:
Nice job bringing this pattern into learner portal 👍 

(See other comment regarding possible suggestions for this query key factory.)